src/nikhilMJun20/MissingNumberFromArray.java

```diff
@@ -0,0 +1,27 @@
+/*Find the missing number in integer array of 1 to 10?
+Hint: You have given an integer array which contains numbers from 1 to 10 but one number is missing, you need to write a Java program to find that missing number in an array.
+*/
+
+package nikhilMJun20;
+
+public class MissingNumberFromArray {
+	void findMissing(int[] intArray) {
+		int originalSum=0, arraySum=0;
+		
+		System.out.println("Given Array : {1, 2, 3, 4, 6, 7, 8, 9, 10}");
+		for(int index=0; index<intArray.length; index++) {
+			arraySum += intArray[index];
+		}
+		
+		for(int num=1; num<=10; num++) {
+			originalSum += num;
```

Comment:
For originalSum you can use math operation as well. 
void findMissingNumber2(int[] array) {
		int al = array.length+1;
		int expectedSum = (al*(al+1))/2;
		int actualSum = 0;
		for(int index=0; index<al-1; index++) {
			actualSum = actualSum + array[index];
		}
		System.out.println("expectedSum "+expectedSum);
		System.out.println("Missing number in given array is: "+(expectedSum-actualSum));
	}

So using one for loop you can achieve the output.

Reply:
Yes... This is a cleaner logic.
Thanks!